### PR TITLE
Create faq.rst

### LINKED ISSUE #281: pyOpenMS docs doesn't have an FAQ page
Currently, the pyOpenMS docs doesn't have an FAQ page. The FAQs currently reside in the [wiki](https://github.com/OpenMS/OpenMS/wiki/Developer-FAQ). I think the pyOpen FAQs from this section should be moved to pyOpenMS docs.

### PR DESCRIPTION
**Pull request by greengypsy**
Adding faq from wiki to pyopenms-docs.

Closes #281.

### CODE REVIEW

File: docs/source/faq.rst [hunk 1]
```diff
@@ -0,0 +1,22 @@
+**************************
+Frequently Asked Questions
+**************************
+
+.. contents:: The following is a list of common questions asked about pyOpenMS.
+
+How can I wrap a new method with pyOpenMS?
+###########################################
```
Comment (tapaswenipathak) [inline docs/source/faq.rst:8]: Let's follow existing formatting style guide for pyopenms-docs `.rst`:
Screenshot 2022-05-24 at 4 09 30 PM

File: docs/source/faq.rst [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+
+Add an entry to ``src/pyOpenMS/pxds/CLASS_NAME.pxd`` with the signature of your new method(s).
+
+
+How can I wrap a new class with pyOpenMS?
+#########################################
+
+Create a new file ``src/pyOpenMS/pxds/CLASS_NAME.pxd`` and use the `procedure outlined <https://github.com/OpenMS/OpenMS/wiki/pyOpenMS#--wrapping-classes>`_.
```
Comment (tapaswenipathak) [inline docs/source/faq.rst:16]: instead of wiki, let's link to the page on pyopenms.readthedocs.

File: docs/source/faq.rst [hunk 3]
```diff
@@ -0,0 +1,22 @@
+
```
Comment (tapaswenipathak) [inline docs/source/faq.rst:1]: let's delete this newline.

File: docs/source/faq.rst (hunk 2 above)
Comment (greengypsy) [reply docs/source/faq.rst:16]: done

File: docs/source/faq.rst (hunk 3 above)
Comment (greengypsy) [reply docs/source/faq.rst]: done

File: docs/source/faq.rst [hunk 4]
```diff
@@ -0,0 +1,21 @@
...
+How can I wrap a new method with pyOpenMS?
+------------------------------------------
+
+Add an entry to ``src/pyOpenMS/pxds/CLASS_NAME.pxd`` with the signature of your new method(s).
+
+
+How can I wrap a new class with pyOpenMS?
+-----------------------------------------
+
+Create a new file ``src/pyOpenMS/pxds/CLASS_NAME.pxd`` and use the `procedure outlined <https://pyopenms.readthedocs.io/en/latest/wrap_classes.html?#how-pyopenms-wraps-python-classes>`_.
```
Comment (tapaswenipathak) [inline docs/source/faq.rst:15]: i was not clear: let's link it to: wrap_classes.rst#how-to-wrap-new-classes

Comment (greengypsy) [reply docs/source/faq.rst]: done

**Review by tapaswenipathak — APPROVED** [review]

**Comment by tapaswenipathak** [comment]
Missed: requires an entry here: https://github.com/OpenMS/pyopenms-docs/blob/master/docs/source/index.rst

**Pull request merged**